CMakeLists.txt

```diff
@@ -60,10 +60,12 @@ set(FlatHash_SRCS
 )
 
 set(FlatBuffers_Tests_SRCS
+  # Until we can use target_compile_definitions to define FLATBUFFERS_TRACK_VERIFIER_BUFFER_SIZE macro
```

Comment:
This should not be the case anymore, so can you revert all changes to this file?